build.bash

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+nproc_default=4
+
+script_name=$(basename "${0}")
+
+show_help() {
+    cat << EOF
+Usage: ./$script_name [OPTIONS]
+
+Build script wrapper around CMake. Supplied arguments that do not match the
+options below will be passed to CMake when generating the build system.
+
+Options:
+      --clean   Delete build directory before invoking CMake.
+      --mpi     Compile MPI executable.
+      --compiler <compiler>
+                Specify the compiler to use.
+  -j <jobs>     Specify the number of parallel jobs in the compilation. By
+                default this value is set to $nproc_default.
+  -h, --help    Show this screen.
+
+Enabling debug mode:
+
+  The release build is default. To enable debug mode, specify the CMake option
+  -DCMAKE_BUILD_TYPE=Debug when invoking $script_name.
+
+Enabling verbose output from Makefile builds:
+
+  To enable more verbose output from Makefile builds, specify the CMake option
+  -DCMAKE_VERBOSE_MAKEFILE=ON when invoking $script_name.
+
+EOF
+}
+
+cmake_args=(-DCMAKE_BUILD_TYPE=Release)
+
+# Argument parsing adapted and stolen from http://mywiki.wooledge.org/BashFAQ/035#Complex_nonstandard_add-on_utilities
+while [ $# -gt 0 ]; do
+    case $1 in
+        --clean)
+            rm -r build
```

Comment:
This is counter-intuitive. I just wanted to clean the run, i.e. removing the directories `bin` and `build`, something like `make clean` but it removed "only" `build` and started compiling.

Reply:
This behaviour is also common to the main branch so I have made a new issue: https://github.com/CABLE-LSM/CABLE/issues/292